application-numbered-content-headings/application-numbered-content-headings-api/pom.xml

```diff
@@ -66,7 +66,6 @@
     <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
-      <scope>test</scope>
```

Comment:
Is this really necessary?

Reply:
yes, otherwise the access to the request does not compile.